kcauto.sikuli/combat.sikuli/lbas.py

```diff
@@ -146,28 +146,40 @@ def _check_and_manage_lbas_fatigue(self, fatigue, group):
                 group_fatigue['medium']
                 if group_fatigue['medium'] else fatigue['medium'])
         else:
+            Util.log_msg("LBAS group #{} has good morale.".format(group))
             lbas_group_nodes_key = 'lbas_group_{}_nodes'.format(group)
             # put LBAS group into air defense mode if it is active but no nodes
             # are assigned to it
             group_sortie_mode = (
                 'sortie'
                 if self.config.combat[lbas_group_nodes_key] else 'defense')
+            Util.log_msg("Assign LBAS group #{a} to {b} mode.".format(a=group, b=group_sortie_mode))
```

Comment:
Linting error. Let's remove the named fields if they're going to be something as vague as `a` and `b`; I don't think they help in readability at all, and it's only two fields.